python-main.js

```diff
@@ -1508,11 +1508,20 @@ function web_editor(config) {
             REPL = null;
             $("#repl").empty();
 
+            var connectTimeout = setTimeout(function() {
+                var error = {"name": "Connection Error", "message": config["translate"]["webusb"]["err"]["reconnect-microbit"]};
+                webusbErrorHandler(error);
```

Comment:
We should probably create a different error for the timeout, as it will help identify it.